messaging/src/test/java/org/springframework/security/messaging/handler/invocation/reactive/CurrentSecurityContextArgumentResolverTests.java

```diff
@@ -186,6 +223,25 @@ private MethodParameter arg0(String methodName) {
 
 	}
 
+	@Target({ ElementType.PARAMETER })
+	@Retention(RetentionPolicy.RUNTIME)
+	@CurrentSecurityContext
+	@interface CurrentAuthentication2 {
```

Comment:
Will you please give this and the other `CurrentAuthentication2` meaningful names? Maybe `AliasedCurrentSecurityContext`. Also, since it appears to only involve `SecurityContext`, it probably shouldn't be called `CurrentAuthentication`.

Reply:
Done